runtime/src/chpl-privatization.c

```diff
@@ -71,8 +72,22 @@ void chpl_newPrivatizedClass(void* v, int64_t pid) {
 }
 
 
-extern void* chpl_getPrivatizedClass(int64_t i) {
+void* chpl_getPrivatizedClass(int64_t i) {
   return chpl_privateObjects[i];
 }
 
 
+void chpl_clearPrivatizedClass(int64_t i) {
+  chpl_privateObjects[i] = NULL;
```

Comment:
It seems like this should be grabbing the `privatizationSync` lock, unless you're positive it can't be called while concurrent calls to `chpl_newPrivatizedClass` are happening. It seems unlikely that you can gaurentee that, but if you can this deserves a comment mentioning why the lock isn't needed